CoreFindView/src/au/gov/asd/tac/constellation/views/find/gui/FindCriteriaPanel.java

```diff
@@ -112,6 +112,7 @@ public FindCriteriaPanel(final FindTopComponent parent, final FindRule localStat
         createCriteriaPanels();
 
         this.localState = localState;
+        this.localState = this.getState();
```

Comment:
This certainly isn't your fault but seeing these 2 lines makes me think there's some function logic that needs to be modified as this looks like it is instantiating the same field twice which looks a little pointless (in reality the 2nd call is actually just updating the existing value)

Reply:
100%, working with it some stuff to just feels wrong? Like things seem to get called more often then they should and things like that. But it's the same with the analytic view so I don't know if its indicative of an issue or just a quirk of how views work, I haven't spent enough time with them.

I'd like to spend some proper time going through it but that will take weeks.